Exec/radiation_tests/Rad2Tshock/bc_fill_3d.F90

```diff
@@ -52,13 +52,13 @@ subroutine ca_hypfill(adv,adv_l1,adv_l2,adv_l3,adv_h1,adv_h2,adv_h3, &
        call eos(eos_input_rt, eos_state)
 
        eint1 = rho1 * eos_state % e
-       etot1 = eint1 + 0.5*rho1*v1**2         
+       etot1 = eint1 + 0.5*rho1*v1**2
     end if
 
-    do n = 1,NVAR
-       call filcc(adv(adv_l1,adv_l2,adv_l3,n),adv_l1,adv_l2,adv_l3,adv_h1,adv_h2,adv_h3, &
-                  domlo,domhi,delta,xlo,bc(1,1,n))
-    enddo
+    !do n = 1,NVAR
```

Comment:
Since this looks to be the same code block as in the conditionals below, to reduce redundancy, can we uncomment this and then remove it from each of the cases below?